### PR TITLE
Implement first time contributor message

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #995 

#### Summary of Changes

This implements a messages for first times and first time contributors.

#### Testing Instructions

Open a PR as "first timer" or first time contributor

#### Todo

- [ ] Better Texts to post

cc @brianteeman ;)

### CODE REVIEW

**Comment by brianteeman** [comment]
> Thank you so much for spending time creating your first pull request for Joomla! We really appreciate the time you spent creating it and  hope that this will be the first of many.

> It will now be reviewed and tested so please make sure you keep an eye on your email for any notifications about its progress. For example there might be some code style changes requested.

**Comment by franz-wohlkoenig** [comment]
> It will now be reviewed and tested

"It will now be reviewed and needs 2 Tests..." or a Link so People can see what a PR has to get before it can merged.

**Comment by brianteeman** [comment]
a link to something that already exists? if so what is the link

**Comment by franz-wohlkoenig** [comment]
https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests#And_now.3F - last Paragraph `If a Pull Request is successfully tested twice, a moderator will add the label 'RTC' to it. RTC means: Ready To Commit'. In basic language it tells someone who has commit rights: This Pull request is successfully tested, and can be added to Joomla!. The admin will add (Merge) the project to the Joomla! CMS github repository. Your Pull Request is definitive now, and will be present in the next version of Joomla! if it was about fixing a bug or in the next minor version if the PR was about a new feature.`

**Comment by brianteeman** [comment]
i personally prefer that all the text is in the comment and not an external link

**Comment by brianteeman** [comment]
> Thank you so much for spending time creating your first pull request for Joomla! We really appreciate the time you spent creating it and hope that this will be the first of many.

> It will now be reviewed and tested so please make sure you keep an eye on your email for any notifications about its progress. For example there might be some code style changes requested.

> More details at https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests#And_now.3F

**Comment by franz-wohlkoenig** [comment]
Text in Comment or external Link like you want. My Goal was to avoid common Questions.

**Comment by brianteeman** [comment]
I think this is now all covered. enough info in the text and more info in the link if required

**Comment by zero-24** [comment]
Thank you so much for spending time creating your first pull request for Joomla! We really appreciate the time you spent creating it and hope that this will be the first of many.<br><br>It will now be reviewed and tested so please make sure you keep an eye on your email for any notifications about its progress. For example there might be some code style changes requested.<br><br>More details <a target="_blank" href="https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests#And_now.3F">in our dokumentation</a>

**Comment by brianteeman** [comment]
Documentation with a c :)

**Comment by brianteeman** [comment]
You have two links in the code and the sentence should end with a .

File: src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php [hunk 1]
```diff
@@ -339,6 +339,13 @@ public function onPullAfterCreate(Event $event)
 			// Check the Categories based on the files that gets changed
 			$this->checkCategories($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
 		}
+
+		// Detect first time contributiors...
+		if ($hookData->pull_request->author_association === 'FIRST_TIME_CONTRIBUTOR' || $hookData->pull_request->author_association === 'FIRST_TIMER')
```
Comment (b2z) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: I cannot find `author_association` property in [Pulls v3 API](https://developer.github.com/v3/pulls/). Am I missing something?

**Comment by zero-24** [comment]
```
{
  "action": "synchronize",
  "number": 997,
  "pull_request": {
    "url": "https://api.github.com/repos/joomla/jissues/pulls/997",
    "id": 153372291,
    "html_url": "https://github.com/joomla/jissues/pull/997",
    "diff_url": "https://github.com/joomla/jissues/pull/997.diff",
    "patch_url": "https://github.com/joomla/jissues/pull/997.patch",
    "issue_url": "https://api.github.com/repos/joomla/jissues/issues/997",
    "number": 997,
    "state": "open",
    "locked": false,
    "title": "Implement first time contributor message",
    "user": {
      "login": "zero-24",
      "id": 2596554,
      "avatar_url": "https://avatars2.githubusercontent.com/u/2596554?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zero-24",
      "html_url": "https://github.com/zero-24",
      "followers_url": "https://api.github.com/users/zero-24/followers",
      "following_url": "https://api.github.com/users/zero-24/following{/other_user}",
      "gists_url": "https://api.github.com/users/zero-24/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/zero-24/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/zero-24/subscriptions",
      "organizations_url": "https://api.github.com/users/zero-24/orgs",
      "repos_url": "https://api.github.com/users/zero-24/repos",
      "events_url": "https://api.github.com/users/zero-24/events{/privacy}",
      "received_events_url": "https://api.github.com/users/zero-24/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": "Pull Request for Issue #995 \r\n\r\n#### Summary of Changes\r\n\r\nThis implements a messages for first times and first time contributors.\r\n\r\n#### Testing Instructions\r\n\r\nOpen a PR as \"first timer\" or first time contributor\r\n\r\n#### Todo\r\n\r\n- [ ] Better Texts to post\r\n\r\ncc @brianteeman ;)",
    "created_at": "2017-11-17T19:12:00Z",
    "updated_at": "2017-11-18T23:33:44Z",
    "closed_at": null,
    "merged_at": null,
    "merge_commit_sha": "86837ea53d115b611363d14154373bc4710e3bb6",
    "assignee": null,
    "assignees": [

    ],
    "requested_reviewers": [

    ],
    "milestone": null,
    "commits_url": "https://api.github.com/repos/joomla/jissues/pulls/997/commits",
    "review_comments_url": "https://api.github.com/repos/joomla/jissues/pulls/997/comments",
    "review_comment_url": "https://api.github.com/repos/joomla/jissues/pulls/comments{/number}",
    "comments_url": "https://api.github.com/repos/joomla/jissues/issues/997/comments",
    "statuses_url": "https://api.github.com/repos/joomla/jissues/statuses/bc3db3ae9fe5314232af74fce8a5795d55302e10",
    "head": {
      "label": "zero-24:commentOnFirstContribution",
      "ref": "commentOnFirstContribution",
      "sha": "bc3db3ae9fe5314232af74fce8a5795d55302e10",
      "user": {
        "login": "zero-24",
        "id": 2596554,
        "avatar_url": "https://avatars2.githubusercontent.com/u/2596554?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/zero-24",
        "html_url": "https://github.com/zero-24",
        "followers_url": "https://api.github.com/users/zero-24/followers",
        "following_url": "https://api.github.com/users/zero-24/following{/other_user}",
        "gists_url": "https://api.github.com/users/zero-24/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/zero-24/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/zero-24/subscriptions",
        "organizations_url": "https://api.github.com/users/zero-24/orgs",
        "repos_url": "https://api.github.com/users/zero-24/repos",
        "events_url": "https://api.github.com/users/zero-24/events{/privacy}",
        "received_events_url": "https://api.github.com/users/zero-24/received_events",
        "type": "User",
        "site_admin": false
      },
      "repo": {
        "id": 89029211,
        "name": "jissues",
        "full_name": "zero-24/jissues",
        "owner": {
          "login": "zero-24",
          "id": 2596554,
          "avatar_url": "https://avatars2.githubusercontent.com/u/2596554?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/zero-24",
          "html_url": "https://github.com/zero-24",
          "followers_url": "https://api.github.com/users/zero-24/followers",
          "following_url": "https://api.github.com/users/zero-24/following{/other_user}",
          "gists_url": "https://api.github.com/users/zero-24/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/zero-24/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/zero-24/subscriptions",
          "organizations_url": "https://api.github.com/users/zero-24/orgs",
          "repos_url": "https://api.github.com/users/zero-24/repos",
          "events_url": "https://api.github.com/users/zero-24/events{/privacy}",
          "received_events_url": "https://api.github.com/users/zero-24/received_events",
          "type": "User",
          "site_admin": false
        },
        "private": false,
        "html_url": "https://github.com/zero-24/jissues",
        "description": "Issue tracking application extending GitHub's issues and pull requests for the Joomla! project.",
        "fork": true,
        "url": "https://api.github.com/repos/zero-24/jissues",
        "forks_url": "https://api.github.com/repos/zero-24/jissues/forks",
        "keys_url": "https://api.github.com/repos/zero-24/jissues/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/zero-24/jissues/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/zero-24/jissues/teams",
        "hooks_url": "https://api.github.com/repos/zero-24/jissues/hooks",
        "issue_events_url": "https://api.github.com/repos/zero-24/jissues/issues/events{/number}",
        "events_url": "https://api.github.com/repos/zero-24/jissues/events",
        "assignees_url": "https://api.github.com/repos/zero-24/jissues/assignees{/user}",
        "branches_url": "https://api.github.com/repos/zero-24/jissues/branches{/branch}",
        "tags_url": "https://api.github.com/repos/zero-24/jissues/tags",
        "blobs_url": "https://api.github.com/repos/zero-24/jissues/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/zero-24/jissues/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/zero-24/jissues/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/zero-24/jissues/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/zero-24/jissues/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/zero-24/jissues/languages",
        "stargazers_url": "https://api.github.com/repos/zero-24/jissues/stargazers",
        "contributors_url": "https://api.github.com/repos/zero-24/jissues/contributors",
        "subscribers_url": "https://api.github.com/repos/zero-24/jissues/subscribers",
        "subscription_url": "https://api.github.com/repos/zero-24/jissues/subscription",
        "commits_url": "https://api.github.com/repos/zero-24/jissues/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/zero-24/jissues/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/zero-24/jissues/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/zero-24/jissues/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/zero-24/jissues/contents/{+path}",
        "compare_url": "https://api.github.com/repos/zero-24/jissues/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/zero-24/jissues/merges",
        "archive_url": "https://api.github.com/repos/zero-24/jissues/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/zero-24/jissues/downloads",
        "issues_url": "https://api.github.com/repos/zero-24/jissues/issues{/number}",
        "pulls_url": "https://api.github.com/repos/zero-24/jissues/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/zero-24/jissues/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/zero-24/jissues/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/zero-24/jissues/labels{/name}",
        "releases_url": "https://api.github.com/repos/zero-24/jissues/releases{/id}",
        "deployments_url": "https://api.github.com/repos/zero-24/jissues/deployments",
        "created_at": "2017-04-21T22:44:33Z",
        "updated_at": "2017-04-21T22:44:36Z",
        "pushed_at": "2017-11-18T23:33:43Z",
        "git_url": "git://github.com/zero-24/jissues.git",
        "ssh_url": "git@github.com:zero-24/jissues.git",
        "clone_url": "https://github.com/zero-24/jissues.git",
        "svn_url": "https://github.com/zero-24/jissues",
        "homepage": "https://issues.joomla.org",
        "size": 20987,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": "JavaScript",
        "has_issues": false,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "open_issues_count": 0,
        "forks": 0,
        "open_issues": 0,
        "watchers": 0,
        "default_branch": "master"
      }
    },
    "base": {
      "label": "joomla:master",
      "ref": "master",
      "sha": "fb88f47799d694b944d55e5a3b3ae0b605ea5c82",
      "user": {
        "login": "joomla",
        "id": 751633,
        "avatar_url": "https://avatars3.githubusercontent.com/u/751633?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/joomla",
        "html_url": "https://github.com/joomla",
        "followers_url": "https://api.github.com/users/joomla/followers",
        "following_url": "https://api.github.com/users/joomla/following{/other_user}",
        "gists_url": "https://api.github.com/users/joomla/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/joomla/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/joomla/subscriptions",
        "organizations_url": "https://api.github.com/users/joomla/orgs",
        "repos_url": "https://api.github.com/users/joomla/repos",
        "events_url": "https://api.github.com/users/joomla/events{/privacy}",
        "received_events_url": "https://api.github.com/users/joomla/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 6099080,
        "name": "jissues",
        "full_name": "joomla/jissues",
        "owner": {
          "login": "joomla",
          "id": 751633,
          "avatar_url": "https://avatars3.githubusercontent.com/u/751633?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/joomla",
          "html_url": "https://github.com/joomla",
          "followers_url": "https://api.github.com/users/joomla/followers",
          "following_url": "https://api.github.com/users/joomla/following{/other_user}",
          "gists_url": "https://api.github.com/users/joomla/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/joomla/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/joomla/subscriptions",
          "organizations_url": "https://api.github.com/users/joomla/orgs",
          "repos_url": "https://api.github.com/users/joomla/repos",
          "events_url": "https://api.github.com/users/joomla/events{/privacy}",
          "received_events_url": "https://api.github.com/users/joomla/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "private": false,
        "html_url": "https://github.com/joomla/jissues",
        "description": "Issue tracking application extending GitHub's issues and pull requests for the Joomla! project.",
        "fork": false,
        "url": "https://api.github.com/repos/joomla/jissues",
        "forks_url": "https://api.github.com/repos/joomla/jissues/forks",
        "keys_url": "https://api.github.com/repos/joomla/jissues/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/joomla/jissues/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/joomla/jissues/teams",
        "hooks_url": "https://api.github.com/repos/joomla/jissues/hooks",
        "issue_events_url": "https://api.github.com/repos/joomla/jissues/issues/events{/number}",
        "events_url": "https://api.github.com/repos/joomla/jissues/events",
        "assignees_url": "https://api.github.com/repos/joomla/jissues/assignees{/user}",
        "branches_url": "https://api.github.com/repos/joomla/jissues/branches{/branch}",
        "tags_url": "https://api.github.com/repos/joomla/jissues/tags",
        "blobs_url": "https://api.github.com/repos/joomla/jissues/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/joomla/jissues/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/joomla/jissues/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/joomla/jissues/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/joomla/jissues/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/joomla/jissues/languages",
        "stargazers_url": "https://api.github.com/repos/joomla/jissues/stargazers",
        "contributors_url": "https://api.github.com/repos/joomla/jissues/contributors",
        "subscribers_url": "https://api.github.com/repos/joomla/jissues/subscribers",
        "subscription_url": "https://api.github.com/repos/joomla/jissues/subscription",
        "commits_url": "https://api.github.com/repos/joomla/jissues/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/joomla/jissues/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/joomla/jissues/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/joomla/jissues/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/joomla/jissues/contents/{+path}",
        "compare_url": "https://api.github.com/repos/joomla/jissues/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/joomla/jissues/merges",
        "archive_url": "https://api.github.com/repos/joomla/jissues/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/joomla/jissues/downloads",
        "issues_url": "https://api.github.com/repos/joomla/jissues/issues{/number}",
        "pulls_url": "https://api.github.com/repos/joomla/jissues/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/joomla/jissues/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/joomla/jissues/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/joomla/jissues/labels{/name}",
        "releases_url": "https://api.github.com/repos/joomla/jissues/releases{/id}",
        "deployments_url": "https://api.github.com/repos/joomla/jissues/deployments",
        "created_at": "2012-10-06T02:12:43Z",
        "updated_at": "2017-08-20T12:42:00Z",
        "pushed_at": "2017-11-18T23:23:02Z",
        "git_url": "git://github.com/joomla/jissues.git",
        "ssh_url": "git@github.com:joomla/jissues.git",
        "clone_url": "https://github.com/joomla/jissues.git",
        "svn_url": "https://github.com/joomla/jissues",
        "homepage": "https://issues.joomla.org",
        "size": 21016,
        "stargazers_count": 59,
        "watchers_count": 59,
        "language": "JavaScript",
        "has_issues": true,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": false,
        "forks_count": 55,
        "mirror_url": null,
        "archived": false,
        "open_issues_count": 64,
        "forks": 55,
        "open_issues": 64,
        "watchers": 59,
        "default_branch": "master"
      }
    },
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/joomla/jissues/pulls/997"
      },
      "html": {
        "href": "https://github.com/joomla/jissues/pull/997"
      },
      "issue": {
        "href": "https://api.github.com/repos/joomla/jissues/issues/997"
      },
      "comments": {
        "href": "https://api.github.com/repos/joomla/jissues/issues/997/comments"
      },
      "review_comments": {
        "href": "https://api.github.com/repos/joomla/jissues/pulls/997/comments"
      },
      "review_comment": {
        "href": "https://api.github.com/repos/joomla/jissues/pulls/comments{/number}"
      },
      "commits": {
        "href": "https://api.github.com/repos/joomla/jissues/pulls/997/commits"
      },
      "statuses": {
        "href": "https://api.github.com/repos/joomla/jissues/statuses/bc3db3ae9fe5314232af74fce8a5795d55302e10"
      }
    },
    "author_association": "CONTRIBUTOR",
    "merged": false,
    "mergeable": null,
    "rebaseable": null,
    "mergeable_state": "unknown",
    "merged_by": null,
    "comments": 11,
    "review_comments": 0,
    "maintainer_can_modify": true,
    "commits": 4,
    "additions": 92,
    "deletions": 0,
    "changed_files": 2
  },
  "before": "12ab58ff4a18f5d1c602a51dde5025b5e028f5ea",
  "after": "bc3db3ae9fe5314232af74fce8a5795d55302e10",
  "repository": {
    "id": 6099080,
    "name": "jissues",
    "full_name": "joomla/jissues",
    "owner": {
      "login": "joomla",
      "id": 751633,
      "avatar_url": "https://avatars3.githubusercontent.com/u/751633?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/joomla",
      "html_url": "https://github.com/joomla",
      "followers_url": "https://api.github.com/users/joomla/followers",
      "following_url": "https://api.github.com/users/joomla/following{/other_user}",
      "gists_url": "https://api.github.com/users/joomla/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/joomla/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/joomla/subscriptions",
      "organizations_url": "https://api.github.com/users/joomla/orgs",
      "repos_url": "https://api.github.com/users/joomla/repos",
      "events_url": "https://api.github.com/users/joomla/events{/privacy}",
      "received_events_url": "https://api.github.com/users/joomla/received_events",
      "type": "Organization",
      "site_admin": false
    },
    "private": false,
    "html_url": "https://github.com/joomla/jissues",
    "description": "Issue tracking application extending GitHub's issues and pull requests for the Joomla! project.",
    "fork": false,
    "url": "https://api.github.com/repos/joomla/jissues",
    "forks_url": "https://api.github.com/repos/joomla/jissues/forks",
    "keys_url": "https://api.github.com/repos/joomla/jissues/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/joomla/jissues/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/joomla/jissues/teams",
    "hooks_url": "https://api.github.com/repos/joomla/jissues/hooks",
    "issue_events_url": "https://api.github.com/repos/joomla/jissues/issues/events{/number}",
    "events_url": "https://api.github.com/repos/joomla/jissues/events",
    "assignees_url": "https://api.github.com/repos/joomla/jissues/assignees{/user}",
    "branches_url": "https://api.github.com/repos/joomla/jissues/branches{/branch}",
    "tags_url": "https://api.github.com/repos/joomla/jissues/tags",
    "blobs_url": "https://api.github.com/repos/joomla/jissues/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/joomla/jissues/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/joomla/jissues/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/joomla/jissues/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/joomla/jissues/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/joomla/jissues/languages",
    "stargazers_url": "https://api.github.com/repos/joomla/jissues/stargazers",
    "contributors_url": "https://api.github.com/repos/joomla/jissues/contributors",
    "subscribers_url": "https://api.github.com/repos/joomla/jissues/subscribers",
    "subscription_url": "https://api.github.com/repos/joomla/jissues/subscription",
    "commits_url": "https://api.github.com/repos/joomla/jissues/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/joomla/jissues/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/joomla/jissues/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/joomla/jissues/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/joomla/jissues/contents/{+path}",
    "compare_url": "https://api.github.com/repos/joomla/jissues/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/joomla/jissues/merges",
    "archive_url": "https://api.github.com/repos/joomla/jissues/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/joomla/jissues/downloads",
    "issues_url": "https://api.github.com/repos/joomla/jissues/issues{/number}",
    "pulls_url": "https://api.github.com/repos/joomla/jissues/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/joomla/jissues/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/joomla/jissues/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/joomla/jissues/labels{/name}",
    "releases_url": "https://api.github.com/repos/joomla/jissues/releases{/id}",
    "deployments_url": "https://api.github.com/repos/joomla/jissues/deployments",
    "created_at": "2012-10-06T02:12:43Z",
    "updated_at": "2017-08-20T12:42:00Z",
    "pushed_at": "2017-11-18T23:23:02Z",
    "git_url": "git://github.com/joomla/jissues.git",
    "ssh_url": "git@github.com:joomla/jissues.git",
    "clone_url": "https://github.com/joomla/jissues.git",
    "svn_url": "https://github.com/joomla/jissues",
    "homepage": "https://issues.joomla.org",
    "size": 21016,
    "stargazers_count": 59,
    "watchers_count": 59,
    "language": "JavaScript",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 55,
    "mirror_url": null,
    "archived": false,
    "open_issues_count": 64,
    "forks": 55,
    "open_issues": 64,
    "watchers": 59,
    "default_branch": "master"
  },
  "organization": {
    "login": "joomla",
    "id": 751633,
    "url": "https://api.github.com/orgs/joomla",
    "repos_url": "https://api.github.com/orgs/joomla/repos",
    "events_url": "https://api.github.com/orgs/joomla/events",
    "hooks_url": "https://api.github.com/orgs/joomla/hooks",
    "issues_url": "https://api.github.com/orgs/joomla/issues",
    "members_url": "https://api.github.com/orgs/joomla/members{/member}",
    "public_members_url": "https://api.github.com/orgs/joomla/public_members{/member}",
    "avatar_url": "https://avatars3.githubusercontent.com/u/751633?v=4",
    "description": ""
  },
  "sender": {
    "login": "zero-24",
    "id": 2596554,
    "avatar_url": "https://avatars2.githubusercontent.com/u/2596554?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/zero-24",
    "html_url": "https://github.com/zero-24",
    "followers_url": "https://api.github.com/users/zero-24/followers",
    "following_url": "https://api.github.com/users/zero-24/following{/other_user}",
    "gists_url": "https://api.github.com/users/zero-24/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/zero-24/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/zero-24/subscriptions",
    "organizations_url": "https://api.github.com/users/zero-24/orgs",
    "repos_url": "https://api.github.com/users/zero-24/repos",
    "events_url": "https://api.github.com/users/zero-24/events{/privacy}",
    "received_events_url": "https://api.github.com/users/zero-24/received_events",
    "type": "User",
    "site_admin": false
  }
}
```

This is the api request for this pull request here. 

Here is the doc about the meanings: https://developer.github.com/v4/reference/enum/commentauthorassociation/ ;)

File: src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php [hunk 2]
```diff
@@ -869,4 +876,23 @@ protected function checkFilesAndAssignCategory($files)
...
+	 * @param   object  $project   Object containing project data
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	protected function sendFirstTimeContributorMessage($hookData, Github $github, Logger $logger, $project)
+	{
+		$message = 'Thank you so much for spending time creating your first pull request for Joomla! We really appreciate the time you spent creating it and hope that this will be the first of many.<br><br>It will now be reviewed and tested so please make sure you keep an eye on your email for any notifications about its progress. For example there might be some code style changes requested.<br><br>More details <a target="_blank" href="https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests#And_now.3F">in our documentation</a>.';
+
```
Comment (b2z) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: Travis does not like this long text :(

**Comment by b2z** [comment]
> Here is the doc about the meanings: https://developer.github.com/v4/reference/enum/commentauthorassociation/

Ok, did not notice that we are on v4 already :)

**Comment by b2z** [comment]
Thank you guys. Lets try it :)

**Pull request merged**